api/src/modules/proxy/proxy.service.ts

```diff
@@ -23,11 +23,6 @@ export class ProxyService {
    */
 
   proxyTileRequest(request: Request, response: Response) {
-    // Use `originalUrl` here, as the `/api/v1/geoprocessing` is otherwise stripped
-    // away, but we need this because the backend microservice expects that as
-    // part of its routes
-    request.url = request.originalUrl.replace('api/v1/', 'api/v1/geodata/');
-
     return this.server.web(
```

Comment:
these changes are different from the ones I had in my PR. I guess is a change due to the decision of removing 'geodata' from the geoprocessing endpoints?

Reply:
yep, this is my suggestion to drop the prefix (which IIRC was my suggestion to use to start with, sorry about the doing and undoing 😬) from the geoprocessing service and to use basically the same paths as we do on the API "proxy-side".

Originally I wasn't too sure how we'd use the geoprocessing service, so my idea for the `geodata` part of the URLs was to group some endpoints together, but after all everything is pretty much related to geodata here, so I'd suggest to drop it and use identical paths in the API and in the geoprocessing service (unless we need to use different paths for specific future endpoints).

Reply:
makes sense to me!

Reply:
ok, thanks, and sorry about my change of mind 😱